package.json

```diff
@@ -89,7 +89,7 @@
     "child-process-promise": "^2.2.1",
     "ember-cli-babel": "^5.1.7",
     "ember-cli-htmlbars": "^1.1.1",
-    "ember-cli-sass": "7.1.1",
+    "ember-cli-sass": "^7.1.1",
```

Comment:
Want to keep this fixed to `7.1.1`.  Basically, we float stuff we own for the most part, but others we keep fixed

Reply:
@anhuynh @EWhite613  Actually the reason we keep this pinned to `7.1.1` is because version `7.1.1.` includes `ember-cli-babel@^5.1.7` (https://github.com/aexmachina/ember-cli-sass/blob/497ef86a8010d37bdc4ec11342c86bea275fa2cf/package.json#L28) while version `7.1.2` includes `ember-cli-babel@^6.6.0` (https://github.com/aexmachina/ember-cli-sass/blob/b293ec8dea605f81f0cfaae580421cf5a791a1ec/package.json#L28) and this is problematic because

> Ember CLI 2.13.0 introduced a change from using Babel 5.x to 6.x which has introduced issues between the Ember CLI addons and Ember applications world.  Applications or addons running a version of Ember CLI >= 2.13.0 almost always encounter issues consuming addons running Ember CLI <= 2.12.3.  Conversely, applications or addons running a version of Ember CLI <= 2.12.3 almost always encounter issues when consuming addons running Ember CLI >= 2.13.0